src/index.ts

```diff
@@ -109,14 +112,52 @@ export const bootstrapEmpackPackedEnvironment = async (
         });
 
         sharedLibsMap[pkg.name] = getSharedLibs(extractedPackage, '');
-
+        paths[pkg.filename] = formPackagesPaths(extractedPackage, '');
         saveFilesIntoEmscriptenFS(Module.FS, extractedPackage, '');
       })
     );
     await waitRunDependencies(Module);
   }
 
-  return sharedLibsMap;
+  return { sharedLibs: sharedLibsMap, paths: paths };
+};
+
+export interface IRemovePackagesFromEnvOptions {
+  /**
+   * The list of packages which should be removed
+   */
+  removeList: any;
+
+  /**
+   * The Emscripten Module
+   */
+  Module: any;
+
+  paths: { [key: string]: string };
+
+  /**
+   * The logger to use during the bootstrap.
+   */
+  logger?: ILogger;
+}
+
+/**
+ * Removing previously installed files
+ *
+ * @param options
+ * @returns void
+ */
+export const removingFiles = async (
```

Comment:
Small suggestion for naming, to get closer to what already exists in the codebase

```suggestion
export const removePackagesFromEmscriptenFS = async (
```